examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -746,7 +752,7 @@ def collate_fn(examples):
                     raise ValueError(f"Unknown prediction type {noise_scheduler.config.prediction_type}")
 
                 # Predict the noise residual and compute loss
-                model_pred = unet(noisy_latents, timesteps, encoder_hidden_states).sample
+                model_pred = unet(noisy_latents, timesteps, encoder_hidden_states, return_dict=False)[0]
```

Comment:
why change this?

Reply:
`torch.compile()` will throw an error about TensorVariables otherwise.

Reply:
it doesn't here. i just implemented this in ST the other day... didn't change return_dict at all.